pkg/convert/apkbuild/apkbuild.go

```diff
@@ -345,33 +349,45 @@ func (c ApkConvertor) mapconvert() {
 		c.GeneratedMelangeConfig.Package.Scriptlets.Trigger.Script = "FIXME"
 	}
 
-	// if c.Apkbuild.Funcs["build"] != nil {
-	//	// todo lets check the command and add the correct cmake | make | meson mconvert pipelines
-	//	//build := c.Apkbuild.Funcs["build"]
-	//}
-
-	// switch c.Apkbuild.BuilderType {
-	//
-	// case BuilderTypeCMake:
-	//	c.GeneratedMelangeConfig.Pipeline = append(c.GeneratedMelangeConfig.Pipeline, config.Pipeline{Uses: "cmake/configure"})
-	//	c.GeneratedMelangeConfig.Pipeline = append(c.GeneratedMelangeConfig.Pipeline, config.Pipeline{Uses: "cmake/build"})
-	//	c.GeneratedMelangeConfig.Pipeline = append(c.GeneratedMelangeConfig.Pipeline, config.Pipeline{Uses: "cmake/install"})
-	//
-	// case BuilderTypeMeson:
-	//	c.GeneratedMelangeConfig.Pipeline = append(c.GeneratedMelangeConfig.Pipeline, config.Pipeline{Uses: "meson/configure"})
-	//	c.GeneratedMelangeConfig.Pipeline = append(c.GeneratedMelangeConfig.Pipeline, config.Pipeline{Uses: "meson/compile"})
-	//	c.GeneratedMelangeConfig.Pipeline = append(c.GeneratedMelangeConfig.Pipeline, config.Pipeline{Uses: "meson/install"})
-	//
-	// case BuilderTypeMake:
-	c.GeneratedMelangeConfig.Pipeline = append(c.GeneratedMelangeConfig.Pipeline, config.Pipeline{Uses: "autoconf/configure"})
-	c.GeneratedMelangeConfig.Pipeline = append(c.GeneratedMelangeConfig.Pipeline, config.Pipeline{Uses: "autoconf/make"})
-	c.GeneratedMelangeConfig.Pipeline = append(c.GeneratedMelangeConfig.Pipeline, config.Pipeline{Uses: "autoconf/make-install"})
-	c.GeneratedMelangeConfig.Pipeline = append(c.GeneratedMelangeConfig.Pipeline, config.Pipeline{Uses: "strip"})
-
-	//default:
-	//	c.GeneratedMelangeConfig.Pipeline = append(c.GeneratedMelangeConfig.Pipeline, config.Pipeline{Uses: "# FIXME"})
-	//
-	//}
+	if c.Apkbuild.Funcs["build"] != nil {
+		var output bytes.Buffer
+		printer := syntax.NewPrinter()
+		err := printer.Print(&output, c.Apkbuild.Funcs["build"])
+		if err != nil {
+			fmt.Errorf("failed to print build function: %w", err)
+		}
+
+		outputStr := output.String()
+
+		// Define the regular expression to find options in the form of -<option>=<value> or --<option>=<value>
+		re := regexp.MustCompile(`--?\w[-\w]*=[^\\\s]+`)
+
+		// Find all matches
+		matches := re.FindAllString(outputStr, -1)
+
+		// Join matches into a single string separated by space and backslash
+		opts := strings.Join(matches, " \\\n")
```

Comment:
I haven't seen opts being added for the existing pipelines but I still attempted to add here but it uses regex, are we intentionally avoiding it or should we add it? If yes, I can refactor it to be used by other pipelines too

Reply:
Good question -- I wonder if some of these opts would overlap with the existing pipeline inputs? It might make sense to try to map some of these to inputs if we can.